### ISSUE TITLE
Maven plugin doesn't work

### ISSUE THREAD

**Issue by RobertoUa** [scalacenter/bloop#548]
When I run `mvn ch.epfl.scala:maven-bloop_2.12:1.0.0-M10:bloopInstall` 
I get `[ERROR] Could not find goal 'bloopInstall' in plugin ch.epfl.scala:maven-bloop_2.12:1.0.0-M10 among available goals bloop`

**Comment by martin-g** [comment]
Duplicate of https://github.com/scalacenter/bloop/issues/492

**Comment by RobertoUa** [comment]
sorry, I search for the issue but back then I couldn't find it.
I added ScalaCenter repository to get the latest version and now I get this `Unable to parse configuration of mojo ch.epfl.scala:maven-bloop_2.12:1.0.0-M10+186-b7c9e1a5:bloopInstall for parameter charset: Cannot find 'charset' in class bloop.integrations.maven.BloopMojo`

**Comment by jvican** [comment]
Yeah, that’s a known issue too, if you give a reproduction I’ll try to fix it https://github.com/scalacenter/bloop/issues/520
